client/cmd/kcoin/misccmd.go

```diff
@@ -31,9 +42,17 @@ The output of this command is supposed to be machine-readable.
 	}
 )
 
-func version(ctx *cli.Context) error {
+func versionPrint(ctx *cli.Context) error {
 	fmt.Println(strings.Title(clientIdentifier))
 	fmt.Println("Version:", params.Version)
+
+	// print latest version if available
+	finder := version.NewFinder("https://s3.amazonaws.com/releases.kowala.io")
```

Comment:
Can these URLs be specified in the CLI command? We can't centralise the location, and that URL will probably change.

Reply:
yeah, I commented this is temp, but I can change on this PR